superflore/generators/bitbake/run.py

```diff
@@ -91,6 +91,7 @@ def main():
         overlay = RosMeta(
             _repo,
             not args.output_repository_path,
+            branch='superflore/{}'.format(now),
```

Comment:
It was intended that this <DATETIME> be identical to the one which is assigned to `ROS_SUPERFLORE_GENERATION_DATETIME`.

Reply:
Yes, thank you!